pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [  # runtime deps  # https://packaging.python.org/en/latest/guide
   "python-dsv-sdk >= 1.0.4",  # credentials.thycotic_dsv
   "python-tss-sdk >= 1.2.1",  # credentials.thycotic_tss
   "requests",  # credentials.aim, credentials.centrify_vault, credentials.conjur, credentials.hashivault
+  "datetime", # credentials.aws_assume_role
```

Comment:
1) the comments have 2 spaces before `#`, follow the style around
2) I was going to suggest keeping the deps sorted (it looks like only pyyaml is out of order for some reason, but...
3) I don't think this is the right (or needed for that matter) dependency — https://pypi.org/p/DateTime is a Zope project. And looking at the actual usage, I'm pretty sure you're only importing the stdlib `datetime` module: I haven't seen any APIs that aren't present @ https://docs.python.org/3.11/library/datetime.html.

```suggestion
```